test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -116,6 +117,112 @@ def run_test(self, model, input, rtol=1e-3, atol=1e-7, do_constant_folding=False
                               input_names=input_names, output_names=output_names,
                               fixed_batch_size=fixed_batch_size)
 
+    # Export Torchvision models
+
+    def test_alexnet(self):
```

Comment:
We probably can skip some of the tests if they are too slow. May worth to take a look at the test time.

Reply:
It will indeed slow down the execution of the tests. 
Do you think running these tests for only 1 opset version would be a good idea?